compiler/rustc_abi/src/lib.rs

```diff
@@ -1,10 +1,11 @@
+#![cfg_attr(feature = "nightly", feature(generic_nonzero))]
 #![cfg_attr(feature = "nightly", feature(step_trait))]
 #![cfg_attr(feature = "nightly", allow(internal_features))]
 #![cfg_attr(feature = "nightly", doc(rust_logo))]
 #![cfg_attr(feature = "nightly", feature(rustdoc_internals))]
 
 use std::fmt;
-use std::num::{NonZeroUsize, ParseIntError};
+use std::num::{NonZero, ParseIntError};
```

Comment:
If we don't have `feature = "nightly"` above, then we can't use it here.

Reply:
Reverted.